extmod/nimble/modbluetooth_nimble.c

```diff
@@ -1966,4 +1966,7 @@ STATIC int ble_secret_store_delete(int obj_type, const union ble_store_key *key)
 
 #endif // MICROPY_PY_BLUETOOTH_ENABLE_PAIRING_BONDING
 
+MP_REGISTER_ROOT_POINTER(struct _mp_bluetooth_nimble_malloc_t *bluetooth_nimble_memory);
```

Comment:
I think this line should be wrapped in `#if !MICROPY_BLUETOOTH_NIMBLE_BINDINGS_ONLY`

Reply:
done